resources/user-agents/browsers/dolfin/dolfin-1-0-1-5.json

```diff
@@ -28,108 +28,45 @@
-          "device": "Samsung GT-B3410"
-        },
-        {
-          "match": "SAMSUNG-GT-B7610*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-B7610"
-        },
-        {
-          "match": "SAMSUNG-GT-E2550*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-E2550"
-        },
-        {
-          "match": "SAMSUNG-GT-I6410*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-I6410"
-        },
-        {
-          "match": "SAMSUNG-GT-I8320*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-I8320"
-        },
-        {
-          "match": "SAMSUNG-GT-S3370*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S3370"
-        },
-        {
-          "match": "SAMSUNG-GT-S3650*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S3650"
-        },
-        {
-          "match": "SAMSUNG-GT-S5230*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S5230"
-        },
-        {
-          "match": "SAMSUNG-GT-S5560*?Dolfin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S5560"
-        },
-        {
-          "match": "SAMSUNG-GT-S5560*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S5560"
-        },
-        {
-          "match": "SAMSUNG-GT-S7550*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S7550"
-        },
-        {
-          "match": "SAMSUNG-GT-i8000*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-I8000"
-        },
-        {
-          "match": "SAMSUNG-S8000*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S8000"
-        },
-        {
-          "match": "SAMSUNG-S8300*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung GT-S8300"
-        },
-        {
-          "match": "SAMSUNG-SGH-E250*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung SGH-E250"
-        },
-        {
-          "match": "SAMSUNG-SGH-F480*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung SGH-F480"
-        },
-        {
-          "match": "SAMSUNG-SGH-G800*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung SGH-G800"
-        },
-        {
-          "match": "SAMSUNG-SGH-U700*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung SGH-U700"
-        },
-        {
-          "match": "SAMSUNG-SGH-U900*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung SGH-U900"
-        },
-        {
-          "match": "SAMSUNG-SGH-i900*?Dolphin/#MAJORVER#.#MINORVER#*",
-          "device": "Samsung SGH-I900"
+          "match": "#DEVICE#/*Dolfin/#MAJORVER#.#MINORVER#*MIDP*",
+          "devices": {
+            "SAMSUNG-GT-B7722": "Samsung GT-B7722",
+            "SAMSUNG-GT-S5620": "Samsung GT-S5620",
+            "SAMSUNG-SGH-T528g": "Samsung SGH-T528g"
+          }
+        },
+        {
+          "match": "#DEVICE#*?Dolphin/#MAJORVER#.#MINORVER#*",
```

Comment:
I couldn't find any useragents that covered these patterns (or any that started with "Samsung-" and contained "Dolphin". 

Can remove if we want, I left them in just in case, and they don't really get looked at by the parser much since they start with "Samsung".